codegen-client/src/main/kotlin/software/amazon/smithy/rust/codegen/client/smithy/generators/client/FluentClientGenerator.kt

```diff
@@ -532,8 +530,8 @@ private fun baseClientRuntimePluginsFn(
                     featureGatedBlock(BehaviorVersionLatest) {
                         rustTemplate(
                             """
-                            if behavior_version.is_none() {
-                                behavior_version = Some(#{BehaviorVersion}::latest());
+                            if config.behavior_version.is_none() {
+                                config.behavior_version = Some(#{BehaviorVersion}::latest());
                             }
```

Comment:
Hmm, generated crates do have a `behavior-version-latest` feature ([example](https://github.com/awslabs/aws-sdk-rust/blob/main/sdk/s3/Cargo.toml#L180)). I'll file an issue just in case, and we can tackle separately if that's something we want to get fixed.